ibtk/src/boundary/cf_interface/CartCellDoubleLinearCFInterpolation.cpp

```diff
@@ -300,11 +300,11 @@ CartCellDoubleLinearCFInterpolation::computeNormalExtension(Patch<NDIM>& patch,
         const int data_depth = data->getDepth();
         const IntVector<NDIM> ghost_width_to_fill = GHOST_WIDTH_TO_FILL;
         Pointer<CartesianPatchGeometry<NDIM> > pgeom = patch.getPatchGeometry();
-        const Box<NDIM>& patch_box = patch.getBox();
+        const auto patch_box = patch.getBox();
```

Comment:
This is a case where we probably want to use `const auto&` instead of `auto` or `const auto`? This is my biggest concern with `auto` --- we don't want to make copies in places where we don't need them.

Reply:
This is a place where I'm not sure how `auto` works. `patch.getBox()` returns a `const Box<NDIM>&`. Does `auto` not pick up on the `&`?

Changing the script to fix this will be simple. I can redo this commit.

Reply:
Auto implies pointers (i.e., in general, `auto *` and `auto` are the same) but not references.